lib/block-patterns.php

```diff
@@ -302,3 +302,134 @@ function( $current_screen ) {
 		}
 	}
 );
+
+/**
+ * FIXME.
+ *
+ * @since 5.9.0
+ */
+function _register_theme_block_patterns() {
```

Comment:
Why are we prefixing this function's name with `_`?

Reply:
It was a rough way to signal the function as private, at least for the purposes of the proof of concept. We are not very consistent with this, and I'm happy to name the function either way. What we _do_ need to do, regardless, is annotate the function with `@internal` and/or `@access private`.